openedx/core/djangoapps/video_config/tests/test_models.py

```diff
@@ -196,15 +200,63 @@ def test_enable_disable_course_flag(self):
         Ensures that the Video Transcript course specific flag, once enabled for a course, can also be disabled.
         """
         self.verify_enable_disable_course_flag(
-            all_courses_model_class=HLSPlaybackEnabledFlag,
-            course_specific_model_class=CourseHLSPlaybackEnabledFlag
+            all_courses_model_class=VideoTranscriptEnabledFlag,
+            course_specific_model_class=CourseVideoTranscriptEnabledFlag
         )
 
     def test_enable_disable_globally(self):
         """
         Ensures that the Video Transcript flag, once enabled globally, can also be disabled.
         """
         self.verify_enable_disable_globally(
-            all_courses_model_class=HLSPlaybackEnabledFlag,
-            course_specific_model_class=CourseHLSPlaybackEnabledFlag
+            all_courses_model_class=VideoTranscriptEnabledFlag,
+            course_specific_model_class=CourseVideoTranscriptEnabledFlag
+        )
+
+
+@ddt.ddt
+class TestVideoUploadsEnabledByDefault(TestCase, FeatureFlagTestMixin):
+    """
+    Tests the behavior of the flags for Video Uploads Enabled By Default feature.
+    These are set via Django admin settings.
+    """
+
+    @ddt.data(
+        *itertools.product(
+            (True, False),
+            (True, False),
+            (True, False),
+        )
+    )
+    @ddt.unpack
+    def test_video_transcript_feature_flags(self, global_flag, enabled_for_all_courses, enabled_for_course_1):
+        """
+        Tests that Video Uploads Enabled By Default feature flags works correctly on tweaking global flags
```

Comment:
nit:  `Tests that Video Uploads Enabled By Default feature flags` to `Tests that video uploads enabled by default feature flags `

Reply:
need to update all docstrings in this file for the reason above